DeprecationWarning: PY_SSIZE_T_CLEAN will be required for '#' formats

I got the following bug report as [Debian#983149](https://bugs.debian.org/983149):

When running the following lines
```
from ephem import Mars, constellation
m = Mars('1970')
print(constellation(m))
```

one gets the warning
```
DeprecationWarning: PY_SSIZE_T_CLEAN will be required for '#' formats
```

This is Python 3.9.1+.

Hello, we are testing Python 3.10.0+ in Fedora and this is not anymore `DeprecationWarning`. 

~~~~
=================================== FAILURES ===================================
_______________________ FunctionTests.test_constellation _______________________

self = <ephem.tests.test_bodies.FunctionTests testMethod=test_constellation>

    def test_constellation(self):
        oneb = readdb('Orion Nebula,f,5.59,-5.45,2,2000.0,')
        oneb.compute('1999/2/28')
>       self.assertEqual(constellation(oneb), ('Ori', 'Orion'))
E       SystemError: PY_SSIZE_T_CLEAN macro must be defined for '#' formats

tests/test_bodies.py:309: SystemError
~~~~

https://docs.python.org/3.10/whatsnew/3.10.html#id2

The PY_SSIZE_T_CLEAN macro must now be defined to use PyArg_ParseTuple() and Py_BuildValue() formats which use #: es#, et#, s#, u#, y#, z#, U# and Z#. See Parsing arguments and building values and the PEP 353. https://bugs.python.org/issue40943


Downstream report: https://bugzilla.redhat.com/show_bug.cgi?id=1891793

--- a/extensions/_libastro.c
+++ b/extensions/_libastro.c
@@ -1,5 +1,6 @@
 /* Python header files. */
 
+#define PY_SSIZE_T_CLEAN
 #include "Python.h"
 
 #if PY_MAJOR_VERSION == 2

